client/my-sites/post-relative-time-status/index.jsx

```diff
@@ -96,10 +97,17 @@ class PostRelativeTime extends React.PureComponent {
 			statusText = this.props.translate( 'pending review' );
 			statusClassName += ' is-pending';
 		} else if ( status === 'future' ) {
-			const scheduledTime = this.props.moment( this.props.post.date ).fromNow();
+			const moment = this.props.moment;
+			const scheduledDate = moment( this.props.post.date );
+			const now = moment();
+			const scheduledTime = scheduledDate.calendar( null, {
+				sameElse: this.props.translate( 'LL [at] LT', {
+					comment: 'moment.js formatting string',
```

Comment:
I think this comment might not be specific enough for the translators...

```suggestion
					comment: 'moment.js formatting string: LT will be translated to time without seconds (ex "8:30PM'), LL is a full date (ex: "September 4 1986"). [at] will be replaced by `at`',

```

Reply:
Agreed, I've updated it! :)

Reply:
Well, in my eyes `moment.js formatting string` gives translators enough hints that any moment.js parameter can be used, depending on the language, i.e. more than `LT` and `LL` is possible. For example a translator could decide that just `LLL` is a feasible alternative for their language.

Reply:
Not to say that a verbose comment like this is bad, it's definitely helpful, so please leave it like that!